app/main.py

```diff
@@ -2,24 +2,67 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int,
+                 max_amount: int) -> None:
```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Apply this change to all classes